packages/contractkit/src/utils/azure-key-vault-client.ts

```diff
@@ -192,8 +186,13 @@ export class AzureKeyVaultClient {
   private static bufferToBigNumber(input: Buffer): BigNumber {
     return new BigNumber(ensureLeading0x(input.toString('hex')))
   }
-  private static bigNumberToBuffer(input: BigNumber): Buffer {
-    return ethUtil.toBuffer(ensureLeading0x(input.toString(16))) as Buffer
+  private static bigNumberToBuffer(input: BigNumber, size: number): Buffer {
+    let hex = input.toString(16)
```

Comment:
wdyt of a one liner `input.toString(16).padStart(64, '0')`?

Reply:
if you look through my commit history i tried padStart first, but the way contractkit is compiled, it doesnt have support for padStart since it is too new of a javascript.